.gitignore

```diff
@@ -4,4 +4,5 @@ node_modules/
 yarn-error.log
 npm-debug.log
 lib/
-playground/
+dist/
+playground/
```

Comment:
an empty line is needed at the end of all files (I'll add prettier in the future)

Reply:
fixed